### ISSUE TITLE
removeSystem is missing from metatable

### ISSUE THREAD

**Issue by karai17** [bakpakin/tiny-ecs#3]
As the title says, you forgot to re-add removeSystem in your refactoring. I added it back in locally and it works fine.

``` lua
worldMetaTable = {
    __index = {
        add = tiny.add,
        addEntity = tiny.addEntity,
        addSystem = tiny.addSystem,
        remove = tiny.remove,
        removeEntity = tiny.removeEntity,
        removeSystem = tiny.removeSystem, -- oops!
        update = tiny.update,
        clearEntities = tiny.clearEntities,
        clearSystems = tiny.clearSystems,
        getEntityCount = tiny.getEntityCount,
        getSystemCount = tiny.getSystemCount,
        getSystemIndex = tiny.getSystemIndex,
        setSystemIndex = tiny.setSystemIndex
    }
}
```

**Comment by bakpakin** [comment]
Just fixed this and released version 1.1-3. It's now on luarocks.

**Comment by karai17** [comment]
Perfect, thanks!
